screens/chat-screen/home.js

```diff
@@ -196,6 +214,32 @@ export default class Home extends Component {
     });
   }
 
+  getSleepData = async () => {
+     const today = Moment(new Date()).format("MM-DD-YYYY")
+     try {        
+             await AsyncStorage.getItem(today).then(key => {
+             if( key != null){
+             console.log("This is the key", key);
+             console.log("Sleep entry for today",JSON.parse(key));
+             const sleepAttempt = JSON.parse(key).attemptToSleepTime;
+             const wakeUp = JSON.parse(key).wakeUpTime;
+             const sleepdate = new Date(sleepAttempt);
+             const wakedate = new Date(wakeUp);
+             this.setState({sleepAttemptTime: sleepdate});
+             this.setState({wakeUpTime: wakedate}); 
```

Comment:
For future reference, you can just do this
```jsx
this.setState({ 
       sleepAttemptTime: sleepdate,
       wakeUpTime: wakedate });
```
All in one line.

Reply:
Aahh i see .... thank you 😊